extras/run_xdf_split_postproces.py

```diff
@@ -1,7 +1,8 @@
 from neurobooth_os.iout.metadator import get_conn
 from neurobooth_os.iout.split_xdf import create_h5_from_csv
+from neurobooth_os.config import neurobooth_config
 
-conn = get_conn()
+conn = get_conn(neurobooth_config['database']['dbname'])
 # location of the csv file containing path filename and task ID
 dont_split_xdf_fpath = "C:/neurobooth"
```

Comment:
Since Brandon is doing a rewrite of split_xdf, we might end up changing this part of the functionality; let's keep this for now instead of changing it twice.